content/05-expression.md

```diff
@@ -207,10 +208,10 @@ Object declaration begins with an opening curly brace `{` after which `key:value
 
 ```haxe
 {
-	key1:value1,
-	key2:value2,
-	...
-	keyN:valueN
+  key1:value1,
```

Comment:
Let's have `key: value` pairs, with a space after the colon. The text immediately above this should also be changed.

Reply:
Ah, good catch. Thanks.